taskana-adapter-tasklistener-example/pom.xml

```diff
@@ -1,259 +1,194 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <parent>
-        <artifactId>taskana-adapter-parent</artifactId>
-        <groupId>pro.taskana</groupId>
-        <version>0.0.1-SNAPSHOT</version>
-    </parent>
-    <modelVersion>4.0.0</modelVersion>
+        <parent>
+                <artifactId>taskana-adapter-parent</artifactId>
+                <groupId>pro.taskana</groupId>
+                <version>0.0.2-SNAPSHOT</version>
+        </parent>
+        <modelVersion>4.0.0</modelVersion>
+
+        <artifactId>taskana-adapter-tasklistener-example</artifactId>
+
+        <packaging>war</packaging>
+
+        <name>Camunda BPM Process Application</name>
+        <description>A Process Application for [Camunda BPM](http://docs.camunda.org). [The project has been generated by the Maven archetype 'camunda-archetype-ejb-war-7.11.1']</description>
+
+        <properties>
+                <failOnMissingWebXml>false</failOnMissingWebXml>
```

Comment:
why?

Reply:
is not needed, taskana-adapter-tasklistener-example works without web.xml. The taskana-adapter-tasklistener-example is just a sample process application that contains a bpmn process definition in order to be able to do some adapter smoke tests. The forms that are contained in the example were created by camunda's archetype and we don't use them.